packages/runtime/src/lib/constants.ts

```diff
@@ -10,6 +10,7 @@ export const MAX_UINT6 = 63n;
 export const DEFAULT_STACK_ELEM = 0n;
 export const MAX_CONCAT_SIZE = 4096;
 export const ALGORAND_MIN_TX_FEE = 1000;
+export const maxByteArrayLength = 4096;
```

Comment:
Let use right name convention.

Reply:
In the goAlgorand the use `maxStringSize`